pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+markers =
+    ci_vendor: Only run these tests within the CI Vendor environment.
```

Comment:
Mind renaming this marker to `ci_only`? It better describes what it is used for. Also please fix-up the sentence and remove the `vendor`.

Reply:
done